### DISCUSSION TITLE
Deployed in Vercel, but doesnt load CSS files

### DISCUSSION THREAD

**Discussion by Nahu-N-Castro** [solidjs/solid-start#1353] [Q&A]
I am developing with Solid Start, try to deploy with GitHub Pages and Vercel.
I'm looking to deploy, it doesn't matter which one I use.
With Vercel I overwrote the output directory to .output/public (before it was in .output and it gave me a 404 assuming it couldn't find the index.html) after overwriting, I was able to get it to take the index.html but it didn't load the files. CSS files.
With GitHub Pages in the main.yaml of the workflow I do specify the same output, but it shows me the root Readme.
could you help me plz? 🙏

**Comment by ryansolid** [comment]
Are you using presets?... I have a repo with working deployments to most major providers on the branches.. https://github.com/solidjs/solid-hackernews. Maybe see if this helps.
